nautobot/docs/release-notes/version-2.0.md

```diff
@@ -211,3 +211,64 @@ Check out the specific changes documented in the table at [UI and REST API Filte
+- [#2721](https://github.com/nautobot/nautobot/issues/2721) - Collapsed `nautobot.utilities` into `nautobot.core`. Refer to the 2.0 migration guide for details.
+- [#2771](https://github.com/nautobot/nautobot/issues/2771) - Updated `jsonschema` version to `~4.17.0`.
+- [#2788](https://github.com/nautobot/nautobot/issues/2788) - Changed REST framework allowed versions logic to support 1.2-1.5 and 2.0+.
+- [#2803](https://github.com/nautobot/nautobot/issues/2803) - Update `mkdocs-include-markdown-plugin` to `3.9.1`.
+- [#2809](https://github.com/nautobot/nautobot/issues/2809) - Renamed `tag` filter on TenantFilterSet to `tags` same as elsewhere.
+- [#2844](https://github.com/nautobot/nautobot/issues/2844) - Updated development dependency `mkdocstrings-python` to 0.8.0.
+- [#2872](https://github.com/nautobot/nautobot/issues/2872) - Refactored imports in utilities app to follow new code style.
+- [#2883](https://github.com/nautobot/nautobot/issues/2883) - Updated `django-taggit` to `3.1.0`.
+- [#2942](https://github.com/nautobot/nautobot/issues/2942) - Updated `django-tree-queries` to `0.13.0`.
+- [#2943](https://github.com/nautobot/nautobot/issues/2943) - Updated dependency rich to ~12.6.0.
+- [#2955](https://github.com/nautobot/nautobot/issues/2955) - Changed CircuitTermination, Device, PowerPanel, RackGroup, Rack, Prefix, VLANGroup, VLAN, Cluster instances associated with existing Site model instances to use the newly created corresponding locations of LocationType Site.
+- [#2993](https://github.com/nautobot/nautobot/issues/2993) - Implemented initial database backend for Celery task results.
+- [#3027](https://github.com/nautobot/nautobot/issues/3027) - Update dependencies `prometheus-client`, `django-storages`, `drf-spectacular`, `black`, `django-debug-toolbar`, `mkdocstrings`, `mkdocstrings-python`, `pylint`, `requests`, `selenium`, `watchdog`.
+- [#3068](https://github.com/nautobot/nautobot/issues/3068) - Renamed fields on Circuit model: `type` to `circuit_type`, `terminations` to `circuit_terminations`, `termination_a` to `circuit_termination_a`, and `termination_z` to `circuit_termination_z`.
+- [#3068](https://github.com/nautobot/nautobot/issues/3068) - Renamed reverse-relation `circuittermination` to `circuit_terminations` on the CablePath model.
+- [#3068](https://github.com/nautobot/nautobot/issues/3068) - Renamed `group` field to `vlan_group` on VLAN model, renamed `ipaddresses` to `ip_addresses` on Service model.
+- [#3068](https://github.com/nautobot/nautobot/issues/3068) - Renamed `group` field to `tenant_group` on Tenant model.
+- [#3069](https://github.com/nautobot/nautobot/issues/3069) - Renamed foreign key fields and related names in Virtualization and DCIM apps to follow a common naming convention. See v2 upgrade guide for full list of changes.
+- [#3177](https://github.com/nautobot/nautobot/issues/3177) - Updated VLANFactory to generate longer and more "realistic" VLAN names.
```

Comment:
```suggestion
- [#3068](https://github.com/nautobot/nautobot/issues/3068) - Renamed reverse-relation `circuittermination` to `circuit_terminations` on the `CablePath` model.
- [#3068](https://github.com/nautobot/nautobot/issues/3068) - Renamed `group` field to `vlan_group` on VLAN model, renamed `ipaddresses` to `ip_addresses` on `Service` model.
- [#3068](https://github.com/nautobot/nautobot/issues/3068) - Renamed `group` field to `tenant_group` on `Tenant` model.
- [#3069](https://github.com/nautobot/nautobot/issues/3069) - Renamed foreign key fields and related names in Virtualization and DCIM apps to follow a common naming convention. See v2 upgrade guide for full list of changes.
- [#3177](https://github.com/nautobot/nautobot/issues/3177) - Updated `VLANFactory` to generate longer and more "realistic" `VLAN` names.
```

Also we're a bit inconsistent in that we have three specific entries for #3068 but just a summary and "see the upgrade guide" for #3069...

Reply:
I think for #3066 we should do like what we did to #3069. #3068 while a single story seems to be broken out by app. Is that sufficient?

Reply:
Yeah, fair enough.